compiler-plugin/src/main/kotlin/arrow/meta/quotes/MetaExtensions.kt

```diff
@@ -279,7 +261,7 @@ fun Meta.throwExpression(
   quote(match, map) { ThrowExpression(it) }
 
 /**
- * TODO
+ * """someFunction(someParam: ${typeElement}.typeReference)"""
```

Comment:
Not sure if it makes sense to not include context for examples like this and `valueArgument`

Reply:
I think `Parameter` or `ValueArgument` is simpler `param: $typeReference` and then when we show `TypeElement` we need to show how to destructure it's type args and other members. `$name`$(typeArgs)`